versionutils/version_test.go

```diff
@@ -71,12 +65,20 @@ var _ = Describe("Version", func() {
 		It("works", func() {
 			expectResult("v0.1.2", "v0.0.1", true, "")
 			expectResult("v0.0.1", "v0.1.2", false, "")
-			expectResult("v0.0.1", "v0.0.0", false, "Version v0.0.0 is not greater than v0.0.0")
+			expectResult("v0.0.1", "v0.0.0", true, "")
```

Comment:
Could add isGreaterThanOrEqual checks to expectResults

Reply:
good idea, found out I wasn't checking for nil